### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by ptsneves**
Minor fixes for issues detected by GCC12.

### CODE REVIEW

File: src/support/server_stats.c [hunk 1]
```diff
@@ -148,6 +148,8 @@ static const uint32_t nfsv3_optype[NFS_V3_NB_COMMAND] = {
 };
 #endif
 
+#endif
```
Comment (dang) [inline src/support/server_stats.c]: This doesn't seem correct.  The #ifdef/#endif macros are already balanced in that file, and there's already an endif for the _USE_NFS3 around nfsv3_optype.  It seems this change shouldn't compile?

File: src/scripts/git_hooks/install_git_hooks.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```
Comment (dang) [inline src/scripts/git_hooks/install_git_hooks.sh:1]: On which system is this necessary?  On linux systems, /bin/sh is /bin/bash.

Comment (ptsneves) [reply src/scripts/git_hooks/install_git_hooks.sh:1]: Ubuntu for example, where sh is dash. bash is not a POSIX compliant shell.

**Comment by ffilz** [comment]
Please submit this via Gerrithub as per src/CONTRIBUTING_HOWTO.txt

**Comment by ptsneves** [comment]
I put them on a common topic. Let me know if you prefer them as standalone.

https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/542679 Fix possible null pointer dereference
https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/542680 Correct shebang for install_git_hooks

**Comment by ptsneves** [comment]
It seems jenkins had some network issues and verification failed.

**Comment by ffilz** [comment]
> I put them on a common topic. Let me know if you prefer them as standalone.
> 
> https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/542679 Fix possible null pointer dereference https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/542680 Correct shebang for install_git_hooks

That's good. Thanks

**Comment by ffilz** [comment]
> It seems jenkins had some network issues and verification failed.

Don't worry too much about Jenkins. It's very much a work in progress and we haven't had someone maintaining it for years. Someone did help us migrate it to a new platform, but the tests still need some attention and it's low on my priority list.

**Comment by ffilz** [comment]
These have been merged (one was actually a duplicate of another gerrithub patch that was previously merged).

**Pull request closed without merging**